src/core/Tag/Tag.tsx

```diff
@@ -40,16 +40,27 @@ export const Tag = (props: TagProps) => {
 
   return (
     <span
-      className={cx('iui-tag', { 'iui-basic': variant === 'basic' }, className)}
+      className={cx(
+        {
+          'iui-tag-basic': variant === 'basic',
+          'iui-tag': variant === 'default',
+        },
+        className,
+      )}
       {...rest}
     >
-      <span className='iui-label'>{children}</span>
+      {variant === 'default' ? (
+        <span className='iui-tag-label'>{children}</span>
+      ) : (
+        children
```

Comment:
so we previously had `iui-label` even for basic tags? Then we should probably make it consistent again with a css patch release?

Reply:
In css, we didnt have it. only non-basic tags had examples with `iui-label`.
Just in react we were adding additional span for all the tags and because css selector had `:not(iui-basic)` we didnt face any issues.
I have separated label class in my [css pr](https://github.com/iTwin/iTwinUI/commit/0d3608f54efe0b327a4224c9fc7295e1d9029fec#diff-e67fd10966ce7f3a92e622e71a8bc6cca0ba784a83821f468487e64cb86e6f85R50) and now if we add it, some styling gets messed up :/

Reply:
Ok, makes sense. Are you planning to make it consistent again in the future?

Reply:
I dont know. Not now, I guess :/